CommandRepository/commands.md

```diff
@@ -0,0 +1,44 @@
+# SwiftBot Commands
```

Comment:
What's the purpose of this file? These are internal commands, and cannot be documented like this.